### PR TITLE
Add a benchmark on individual Unicode codepoints reading/writing

### PR DESCRIPTION
**Pull request by fzhinkin**

### CODE REVIEW

File: benchmarks/src/commonMain/kotlin/BufferOps.kt [hunk 1]
```diff
@@ -415,3 +416,67 @@ open class IndexOfByteString {
...
     fun benchmark() = buffer.indexOf(byteString)
 }
+
+@State(Scope.Benchmark)
+open class Utf8CodePointsBenchmark : BufferRWBenchmarkBase() {
+    private val codePointsCount = 128
+
+    // Encoding names follow naming from Utf8StringBenchmark
+    @Param("ascii", "utf8", "sparse", "2bytes", "3bytes", "4bytes", "bad")
+    var encoding: String = "ascii"
```
Comment (shanshin) [inline benchmarks/src/commonMain/kotlin/BufferOps.kt:426]: Maybe it's better to use enum in such cases?

File: benchmarks/src/commonMain/kotlin/BufferOps.kt [hunk 2]
```diff
@@ -415,3 +416,67 @@ open class IndexOfByteString {
...
+    @Setup
+    fun fillCodePointsArray() {
+        fun IntArray.fill(generator: () -> Int) {
+            for (idx in this.indices) {
+                this[idx] = generator()
+            }
+        }
+
+        when (encoding) {
+            "ascii" -> codePoints.fill { Random.nextInt(' '.code, '~'.code) }
```
Comment (shanshin) [inline benchmarks/src/commonMain/kotlin/BufferOps.kt:444]: Why are codes less than 0x20 ignored?)

**Review by shanshin — APPROVED** [review]

File: benchmarks/src/commonMain/kotlin/BufferOps.kt (hunk 2 above)
Comment (fzhinkin) [reply benchmarks/src/commonMain/kotlin/BufferOps.kt:444]: Just a personal preference :)

File: benchmarks/src/commonMain/kotlin/BufferOps.kt (hunk 1 above)
Comment (fzhinkin) [reply benchmarks/src/commonMain/kotlin/BufferOps.kt:426]: I decided to use the same values as for the old read/writeString benchmark. I can replace the parameter with enum, but then the old value need to be replaced (especially `2|3|4bytes`), and that would complicate results comparison against older releases.

**Pull request merged**